tests/test_serialization.py

```diff
@@ -412,7 +412,7 @@ class BadSerializer(Schema):
             bad_field = fields.Method("invalid")
 
         u = User("Foo")
-        with pytest.raises(ValueError):
+        with pytest.raises(AttributeError):
             BadSerializer().dump(u)
```

Comment:
Is the call to `dump()` necessary for the error to be raised?

Reply:
Good catch, no `dump()` will not be necessary anymore